posthog/models/user.py

```diff
@@ -108,7 +113,7 @@ class User(AbstractUser, UUIDClassicModel):
     toolbar_mode: models.CharField = models.CharField(
         max_length=200, null=True, blank=True, choices=TOOLBAR_CHOICES, default=TOOLBAR
     )
-
+    events_column_config: JSONField = JSONField(default=events_column_config_default)
```

Comment:
Was refactoring this to it's own `UserPreference` model but decided it was not a good use of time right now, ship fast. If we do add this for other views, or add more personalization / config options we should abstract it to a separate model to avoid regressing to the loading issue.